mne/viz/circle.py

```diff
@@ -122,9 +122,8 @@ def _plot_connectivity_circle_onpick(event, fig=None, axes=None, indices=None,
         fig.canvas.draw()
 
 
-@deprecated("Starting version v0.25, mne.connectivity sub-modules "
-            "will be housed in 'mne-connectivity'. Download it "
-            "by running 'pip install mne-connectivity'.")
+@deprecated(CONNECTIVITY_DEPRECATION_MSG)
+@verbose
```

Comment:
e.g. this function does not take `@verbose` so now test fails.

Reply:
yes you cannot decorate with verbose a function that does not have a verbose parameter